### DISCUSSION TITLE
Pool not recognising Launcher ID intermittently

### DISCUSSION THREAD

**Discussion by porkopops** [Chia-Network/chia-blockchain#7452] [Support]
I'm using pool.garden with the 1.2.1 Windows GUI and I seem to be dropping off the pool.

Here's the link to my launcher ID on the pool: https://pool.garden/farmer/6ff420ad1114f4c3d6edd09c8f491efd8c2d15702c89406051440c8c8475616d

This morning, around 8am, the Launcher ID was not found on the website (unfortunately the last time I checked was Friday so I have no idea how long it was seemingly disconnected). I restarted the Chia GUI and, a while later, I could see my Launcher ID on the pool website again.

I checked around 1630 today and I was not able to find the Launcher ID again. so I restarted the Chia GUI and a while later I could see my Launcher ID on the website.

I have the debug.log set to INFO and I'm seeing activity even when it appears that I'm not on the pool, although I'm not exactly sure what specifically to check for in the log.

Any ideas as to the possible reason why?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
